### PR TITLE
Privacy statement wording update

### PR DESCRIPTION
**Pull request by iteles**
This PR:
+ Updates the privacy statement in line with #716 
+ Adds an anchor tag for 'Consent' ahead of #715

@Danwhy Could you please run this and see if it looks ok as part of your review? 
I updated the deps but couldn't get it to run locally without spending an inordinate amount of time sorting out a few react issues 🙈

### CODE REVIEW

File: src/js/components/privacy-message.js [hunk 1]
```diff
@@ -75,34 +134,65 @@ const PrivacyMessageComponent = () => {
...
+                      data in the ways described in the privacy statement. Consent
+                      is an ongoing process – you are free to withdraw consent at any time.
+                      If you want to withdraw consent, you can email
+                      <a href="mailto:hello@quodl.co.uk">hello@quodl.co.uk</a>.
+                      Depending on the specific aspects of consent you wish to withdraw,
+                      you may not be able to continue to log in to Quodl once consent is
+                      withdrawn. As you can see from the privacy statement, you can
+                      also request a copy of your data, request a correction to
+                      anything that you think is inaccurate, or have all of your
+                      identifying data deleted. For all of these you can just email
```
Comment (Danwhy) [inline src/js/components/privacy-message.js]:
```suggestion
                      identifying data deleted. For all of these you can just email <a href="mailto:hello@quodl.co.uk">hello@quodl.co.uk</a>.
```

File: src/js/components/privacy-message.js [hunk 2]
```diff
@@ -75,34 +134,65 @@ const PrivacyMessageComponent = () => {
...
+                      <a href="mailto:hello@quodl.co.uk">hello@quodl.co.uk</a>.
```
Comment (Danwhy) [inline src/js/components/privacy-message.js]:
```suggestion
```

File: src/js/components/privacy-message.js [hunk 3]
```diff
@@ -75,34 +134,65 @@ const PrivacyMessageComponent = () => {
...
                       <div className="subtitle">
-                          People are sometimes worried about their data being used to make significant decisions
-                          automatically, without human intervention. Quodl may share performance with your institution, but
-                          does not make automated decisions (it shows different feedback based on your performance and
-                          participation, but doesn’t make decisions around assessment or streaming or rewards or
-                          interventions – if that were done it would be with human intervention by your institution).
+                      When you register, you give consent for Quodl to use your
+                      data in the ways described in the privacy statement. Consent
+                      is an ongoing process – you are free to withdraw consent at any time.
+                      If you want to withdraw consent, you can email
```
Comment (Danwhy) [inline src/js/components/privacy-message.js]:
```suggestion
                      If you want to withdraw consent, you can email <a href="mailto:hello@quodl.co.uk">hello@quodl.co.uk</a>.
```

File: src/js/components/privacy-message.js [hunk 4]
```diff
@@ -75,34 +134,65 @@ const PrivacyMessageComponent = () => {
...
+                      <a href="mailto:hello@quodl.co.uk">hello@quodl.co.uk</a>.
```
Comment (Danwhy) [inline src/js/components/privacy-message.js]:
```suggestion
```

File: src/js/components/privacy-message.js [hunk 5]
```diff
@@ -75,34 +134,65 @@ const PrivacyMessageComponent = () => {
                       <div className="subtitle">
                           Yes. You can see virtually all the data we hold when you are logged in: The scores and answers on
                           the questions across the quizzes you have taken. If you would like to request all of the data we hold
-                          that is associated with your email address, email: hello@quodl.co.uk.
+                          that is associated with your email address, email:
```
Comment (Danwhy) [inline src/js/components/privacy-message.js]:
```suggestion
                          that is associated with your email address, email: <a href="mailto:hello@quodl.co.uk">hello@quodl.co.uk</a>.
```

File: src/js/components/privacy-message.js [hunk 6]
```diff
@@ -75,34 +134,65 @@ const PrivacyMessageComponent = () => {
...
+                          <a href="mailto:hello@quodl.co.uk">hello@quodl.co.uk</a>.
```
Comment (Danwhy) [inline src/js/components/privacy-message.js]:
```suggestion
```

**Comment by iteles** [comment]
@Danwhy Thanks for catching! This new suggestion feature is pretty cool!

Assigning back to you for a final review (or I will do, once CircleCI is happy) 😊

**Pull request merged**